types/jest/index.d.ts

```diff
@@ -525,16 +525,21 @@ declare namespace jest {
          * does not contain all of the elements in the expected array. That is,
          * the expected array is not a subset of the received array. It is the
          * inverse of `expect.arrayContaining`.
+         *
+         * Optionally, you can provide a type for the elements via a generic.
          */
-        arrayContaining(arr: any[]): any;
+        arrayContaining<E>(arr: E[]): any;
```

Comment:
```suggestion
        arrayContaining<E = any>(arr: E[]): any;
```

Explicit fallback rather than implicit